app/helpers/authentication_helper.rb

```diff
@@ -0,0 +1,19 @@
+module AuthenticationHelper
```

Comment:
Please rename this to `AuthorizationHelper`. They feel similar but have important differences.